### ISSUE TITLE
DNS resolution broken inside of rancher-desktop VM

### ISSUE THREAD

**Issue by bashofmann** [rancher-sandbox/rancher-desktop#572]
After a clean installation, the DNS resolution is broken inside of the rancher-desktop VM. This prevents external container images to be loaded, for example.

rancher-desktop 0.4.1
on MacOS Big Sur 11.5.2

When I create a shell inside of the VM the `resolv.conf` looks like this:

```
$ cat /etc/resolv.conf
nameserver 192.168.5.3
```

An http request to a publicly available website fails (e.g. `wget http://suse.com`).

`192.168.5.3` seems to be the internal lima-vm DNS server (https://github.com/lima-vm/lima/blob/5619725e6f4505ae144940f127b3151c55bbbd2e/docs/network.md#network). A ping to it works.

If I add a publicly available DNS server:

```
$ cat /etc/resolv.conf
nameserver 192.168.5.3
nameserver 8.8.8.8
```

DNS resolution and requests to publicly available websites work and images can be pulled successfully.

I already tried a factory reset, but that did not resolve the issue.

**Comment by crumohr** [comment]
Same with me, after I added `1.1.1.1` in the macOS network settings for my network adapter it started to work. I guess it's a problem with IPv6 maybe.

**Comment by bashofmann** [comment]
The DNS server on my Mac is the one from my Fritzbox router. Changing my Mac network settings to use `8.8.8.8` as a DNS server actually resolves the problem as well.

**Comment by crumohr** [comment]
Most definitely! Cluster DNS in combination with DNS over IPv6 on macOS is broken.

**Comment by jandubois** [comment]
This should be fixed in by moving to lima 0.6.3+: https://github.com/lima-vm/lima/pull/199
